pkg/build/builder/cmd/builder.go

```diff
@@ -131,7 +132,18 @@ func newBuilderConfigFromEnvironment(out io.Writer, needsDocker bool) (*builderC
 		}
 		istorage.Transport.SetStore(store)
 
-		dockerClient, err := bld.GetDaemonlessClient(systemContext, store, os.Getenv("BUILD_ISOLATION"))
+		// By default, enable a blob cache, but allow it to be moved or
+		// disabled via the environment.
+		cfg.blobCache = "/var/cache/blobs"
+		if dir, isSet := os.LookupEnv("BUILD_BLOBCACHE"); isSet {
+			cfg.blobCache = dir
+		}
+
+		// An empty value implies the system default location, somewhere under
+		// /var/lib/containers/cache.
+		systemContext.BlobInfoCacheDir = os.Getenv("BUILD_BLOBINFOCACHE")
```

Comment:
and name this one BUILD_BLOBINFOCACHE_DIR or something that makes it clear it points to a path.

Reply:
also presumably you need to handle the case that someone enabled the blob cache but did not set this dir.

or do away w/ this env entirely and just hardcode the expected cache directory.

Reply:
Okay, dropping the variable, which should result in containers/image always using its default location.